src/take.js

```diff
@@ -0,0 +1,26 @@
+import { isRegExp, isString, isFunction } from 'lodash'
+
+const take = (match, callback) => (action, store) => {
+  const isMatching = ( // test matching
+    // to a string converted to regexp
+    (
+      isString(match) &&
+      action.type === match
+    )
+    // to a function
+    || (
+      isFunction(match) &&
+      match(action, store)
+    )
+    // to a regexp
+    || (
+      isRegExp(match) &&
+      action.type.match(match)
+    )
+  )
```

Comment:
I think it could be useful if we could give an array of actions to match : 
```es6
take(['ACTION1', 'ACTION2'] console.log),
```

Reply:
#25